hooli_data_eng/resources/sensor_smtp.py

```diff
@@ -64,89 +65,62 @@ def send_email_alert(
 
     if smtp_type == "SSL":
         send_email_via_ssl(
-            email_from, email_password, email_to, message, smtp_host, smtp_port=smtp_port or 465
+            email_from,
+            email_password,
+            email_to,
+            message,
+            smtp_host,
+            smtp_port=smtp_port or 465,
         )
     elif smtp_type == "STARTTLS":
         send_email_via_starttls(
-            email_from, email_password, email_username, email_to, message, smtp_host, smtp_port=smtp_port or 587
+            email_from,
+            email_password,
+            email_username,
+            email_to,
+            message,
+            smtp_host,
+            smtp_port=smtp_port or 587,
         )
     else:
-        raise DagsterInvalidDefinitionError(f'smtp_type "{smtp_type}" is not supported.')
-
-
-class SESEmailAlert():
-    def __init__(
-        self, 
-        smtp_host, 
-        smtp_username, 
-        smtp_password, 
-        smtp_email_from, 
-        smtp_email_to
-    ):
-        self._smtp_host = smtp_host
-        self._smtp_username = smtp_username
-        self._smtp_password = smtp_password
-        self._smtp_email_from = smtp_email_from
-        self._smtp_email_to = smtp_email_to
+        raise DagsterInvalidDefinitionError(
+            f'smtp_type "{smtp_type}" is not supported.'
+        )
+
+
+class SESEmailAlert(ConfigurableResource):
+    smtp_host: str
+    smtp_username: str
+    smtp_password: str
+    smtp_email_from: str
+    smtp_email_to: List[str]
```

Comment:
Think this was causing issues as `list[str]` bc of a Python version <3.9, have to use `typing.List` for now